SimDataFormats/TrackerDigiSimLink/src/PixelSimHitExtraInfo.cc

```diff
@@ -0,0 +1 @@
+#include "SimDataFormats/TrackerDigiSimLink/interface/PixelSimHitExtraInfo.h"
```

Comment:
As far as I can tell, this file serves no purpose. I think you can remove it.

Reply:
@ferencek I produce this file, to follow the logic of the other files in the directory [PixelDigiSimLink](https://github.com/CMSTrackerDPG/cmssw/blob/1f8507ad53be5097978fe149410a5e33765b6b1d/SimDataFormats/TrackerDigiSimLink/src/PixelDigiSimLink.cc) and [StripDigiSimLink](https://github.com/CMSTrackerDPG/cmssw/blob/1f8507ad53be5097978fe149410a5e33765b6b1d/SimDataFormats/TrackerDigiSimLink/src/StripDigiSimLink.cc). But I will see what happens if I remove it...